guides/plugins/plugins/administration/modify-blacklist-for-dynamic-product-groups.md

```diff
@@ -0,0 +1,89 @@
+
+In this Guide you'll get two quick examples on how to either add new properties to this blacklist or even remove
+properties from the blacklist, so they're actually shown in the administration and thus can be used.
+
+## Prerequisites
+
+This guide **will not** explain in detail how to override an existing component.
+For this guide you'll have to extend the component [sw-product-stream-field-select](https://github.com/shopware/platform/blob/v6.3.4.1/src/Administration/Resources/app/administration/src/module/sw-product-stream/component/sw-product-stream-field-select/index.js) though, since it's the one [actually checking for the properties in the computed property options](https://github.com/shopware/platform/blob/v6.3.4.1/src/Administration/Resources/app/administration/src/module/sw-product-stream/component/sw-product-stream-field-select/index.js#L41).
+
+An example on how to override a component can be found [here](./customizing-components.md).
+
+## Adding properties to blacklist
+
+As already mentioned in the prerequisites, the check for properties in the blacklist is done in the computed property `options`.
+Therefore you'll have to make sure your modifications are done **before** the check happens.
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/sw-product-stream-field-select/index.js" %}
+```javascript
+const { Component } = Shopware;
+
+Component.override('sw-product-stream-field-select', {
+    computed: {
+        options() {
+            this.conditionDataProviderService.addToGeneralBlacklist(['deliveryTimeId']);
+            return this.$super('options');
+        }
+    }
+});
+```
+{% endcode %}
+
+This example will simply add the property `deliveryTimeId` to the blacklist, so it's not configurable using the Administration anymore.
+There's also nested properties, so called 'entity properties', which are selectable once you've chosen a property such as `Categories`.
+Those entity properties can also be added to the blacklist by using the method `addToEntityBlacklist` instead:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/sw-product-stream-field-select/index.js" %}
+```javascript
+const { Component } = Shopware;
+
+Component.override('sw-product-stream-field-select', {
+    computed: {
+        options() {
+            this.conditionDataProviderService.addToEntityBlacklist('category', ['breadcrumb']);
+            return this.$super('options');
+        }
+    }
+});
+```
+{% endcode %}
+
+This example would forbid the usage of `breadcrumb` from the `category` entity.
+
+## Removing properties from the blacklist
+
+Most likely you'd want to do the opposite and enable properties by removing entries from the blacklist.
+This can be done exactly like adding properties to the blacklist.
+One method to remove a property from the "general blacklist", which is the first dropdown, or from the "entity blacklist",
```

Comment:
This sentence may be hard to read, so I tried to divide it to get it easier to grap.

```suggestion
* Remove a property from the "general blacklist", which is the first dropdown
```